cloudant-client/src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -257,7 +258,7 @@ public CloudantClient build() {
 
             // Create IAM cookie interceptor and set in HttpConnection interceptors
             IamCookieInterceptor cookieInterceptor = new IamCookieInterceptor(this.iamApiKey,
-                    this.url.toString());
+                    this.url.toString(), proxyURL);
```

Comment:
It doesn't matter because there is no shadowing and sadly there is very little consistency in this file between using `this` or not on the instance fields, but anyway to be consistent with your `CookieInterceptor` change it might be worth adding `this`
```suggestion
                    this.url.toString(), this.proxyURL);
```

Reply:
Fixed in a1c95fad4ccf0e845fe4649613fc184c33168220